csharp/ql/test/resources/stubs/runtime.win-arm64.runtime.native.System.Data.SqlClient.sni/4.4.0/runtime.win-arm64.runtime.native.System.Data.SqlClient.sni.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why are we getting just `.csproj` files, and no `.cs` files? Perhaps @tamasvajk knows?

Reply:
We can remove the folders that contain only a .csproj file, but we need to also remove the reference to these .csproj files.

We probably end up with these projects, because the original assemblies didn't contain any public types. As far as I remember, we're creating the folder structure based on the `project.assets.json` file (and not the extracted types), which includes these assemblies too.

Reply:
Do you prefer that I manually go through the generated stubs and remove what is strictly not needed?

Reply:
In that case let's just leave them in.